src/cdrizzlebox.c

```diff
@@ -553,10 +567,15 @@ do_kernel_lanczos(struct driz_param_t* p) {
                 for (jj = nyi; jj <= nya; ++jj) {
                     for (ii = nxi; ii <= nxa; ++ii) {
                         /* X and Y offsets */
-                        ix = fortran_round(fabs(xx - (double)ii) * lanczos.sdp) + 1;
-                        iy = fortran_round(fabs(yy - (double)jj) * lanczos.sdp) + 1;
-
-                        /* Weight is product of Lanczos function values in X and Y */
+                        ix =
```

Comment:
This odd reformatting is due to the many indentations.  The code would be more readable move some of these indented portions to a separate function, as well as to reduce line length.